compiler/damlc/daml-stdlib-src/LibraryModules.daml

```diff
@@ -9,6 +9,7 @@ daml 1.2
 module LibraryModules where
 
 import DA.Action
+import DA.Action.State
```

Comment:
I forgot that when I added the module which results in it not being proprely packaged (which isn’t required for the doctests to work).